### PR TITLE
Display SubBodies

### PR DESCRIPTION
**Pull request by KevinBoulongne**
Depends on #1447

### CODE REVIEW

File: app/src/main/java/com/infomaniak/mail/data/cache/mailboxContent/DraftController.kt [hunk 1]
```diff
@@ -148,11 +149,14 @@ class DraftController @Inject constructor(
...
         } ?: ""
 
+        val subBodiesContent = message.body?.subBodies?.let(UiUtils::formatSubBodiesContent) ?: ""
+
         return """
             <div id=\"answerContentMessage\" class="${MessageBodyUtils.INFOMANIAK_REPLY_QUOTE_HTML_CLASS_NAME}" >
             <div>$messageReplyHeader</div>
             <blockquote class=\"ws-ng-quote\">
             $previousBody
+            $subBodiesContent
```
Comment (LunarX) [inline app/src/main/java/com/infomaniak/mail/data/cache/mailboxContent/DraftController.kt:159]: Is this where we want to insert subbodies?

File: app/src/main/java/com/infomaniak/mail/utils/UiUtils.kt [hunk 2]
```diff
@@ -130,4 +131,15 @@ object UiUtils {
...
     fun dividerDrawable(context: Context) = AppCompatResources.getDrawable(context, R.drawable.divider)
+
+    fun formatSubBodiesContent(subBodies: List<SubBody>): String {
+        var subBodiesContent = ""
+        subBodies.forEach { subBody ->
+            subBody.bodyValue?.let {
+                if (subBodiesContent.isNotEmpty()) subBodiesContent += "<br/>"
+                subBodiesContent += "<blockquote>${it}</blockquote>"
+            }
+        }
```
Comment (LunarX) [inline app/src/main/java/com/infomaniak/mail/utils/UiUtils.kt:142]: We don't need to somehow encapsulate child subbodies inside their parent?

Comment (KevinBoulongne) [reply app/src/main/java/com/infomaniak/mail/utils/UiUtils.kt:142]: Nope, we do the simpler solution for this one.

File: app/src/main/java/com/infomaniak/mail/data/cache/mailboxContent/DraftController.kt (hunk 1 above)
Comment (KevinBoulongne) [reply app/src/main/java/com/infomaniak/mail/data/cache/mailboxContent/DraftController.kt:159]: Yes.

Comment (KevinBoulongne) [reply app/src/main/java/com/infomaniak/mail/data/cache/mailboxContent/DraftController.kt:159]: Yes?

**Pull request merged**